JWKS URI addition to client secret rotation (#3071)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/client-secret-rotation-key/main/index.md

    > **Note:** You can try this in Postman using the **Add a client secret: auto-generated** request.

    When you generate a new secret, the original secret remains in **Active** status. Both secrets are stored in parallel, allowing clients to continue using the old secret during secret rotation. Any requests for a secret return the newly generated client secret, but any requests that are sent using the previous secret still work until the status of that client secret is set to **Inactive**.

4. [Update your web app](/docs/guides/sign-into-web-app/-/main/#configure-the-package) to start using the newly generated client secret.

5. Test your app and ensure that all functionality works with the newly generated client secret.

6. After you update the client secret, return to the **Client secrets** section for the app and change the old client secret status from **Active** to **Inactive**.

    > **Note:** You can try this in Postman using the **Deactivate a client secret** request.

> 7. Re-validate that your app works after changing the old client secret status to rule out any possibility that your app is using older credentials. Should there be any issues with your app, you can change the status of the old client secret back to **Active** and troubleshoot.

Reviewer: Does it need the hyphen?
```suggestion
7. Revalidate that your app works after changing the old client secret status to rule out any possibility that your app is using older credentials. Should there be any issues with your app, you can change the status of the old client secret back to **Active** and troubleshoot.
```

> 8. After validation is successful, you can then delete the old secret using the Admin Console by changing the status of the old secret from **Inactive** to **Delete**. This ensures that the older secret isn’t used by mistake.

Reviewer: ```suggestion
8. After revalidation is successful, you can then delete the old secret using the Admin Console by changing the status of the old secret from **Inactive** to **Delete**. This ensures that the older secret isn’t used by mistake.
```

### Use a URL to fetch keys dynamically

This option allows you to host your public key in a URI and paste the link to the public key in the Admin Console. This URL contains public keys that clients can use to verify the signature of client-based access tokens and OpenID Connect ID tokens. By hosting the keys in a URL, you can conveniently rotate the keys without having to update the app configuration every time. Okta dynamically fetches the latest public key for the app, which eliminates the need to manually update the public key in the Admin Console when you’re rotating the key pair.

> **Note:** If you switch from saving keys in Okta to using a URL to fetch keys dynamically, any saved public keys are deleted.

1. After you select **Use a URL to fetch keys dynamically**, enter the URL where you are hosting your public key in the **URL** box, for example: `https:/${yourOktaDomain}/oauth2/v1/keys`.

    > **Note:** You can try this in Postman using the **Add a JWK URI** request.

2. Click **Save**.

## JWK management

> To add more keys, deactivate a key, and delete a key, follow these steps:

Reviewer: Doesn't "and" imply that the steps show how to do all three together?
```suggestion
To add more keys, deactivate a key, or delete a key, follow these steps:
```
